docs/architecture/adr-050-sign-mode-textual.md

```diff
@@ -56,17 +57,35 @@ Ledger devices have limited character display capabilities, so all strings MUST
 
 In particular, the newline `"\n"` (ASCII: 10) character is forbidden.
 
-### 3. Strings SHOULD have the `<key>: <value>` format
+### 3. Strings MUST match the `^\*?(>* )?((?![\*>: ]).+: )?((?![\*>: ]).+)$` Regex
```

Comment:
@JimLarson Trying to word out the details we discussed about `*` `>` and ` `, I believe a Regex is way clearer.

Reply:
What's the meaning of having multiple asterix (`*`) in the front? `\*`?

Reply:
You can't have multiple asterisks in the front with this Regex, see the examples of invalid strings.

Reply:
Regexes can be problematic because different regex engines have different interpretations of the same regex expression. They are also not particularly human-friendly :) Normally things like this are specified in Backhaus-Naur Form (BNF), [here's a random example](https://datatracker.ietf.org/doc/html/rfc7233#section-2.1) from HTTP.